src/routes/transactions.rs

```diff
@@ -84,7 +84,7 @@ pub fn queued_transactions(
     timezone_offset: Option<String>,
     trusted: Option<bool>,
 ) -> ApiResult<content::Json<String>> {
-    context
+    let resp = context
```

Comment:
Is this assignment necessary? Given the return statement, maybe we don't need it.